Tests/DependencyInjection/AbstractMongoDBExtensionTest.php

```diff
@@ -409,7 +409,14 @@ public function testResolveTargetDocument()
 
         $definition = $container->getDefinition('doctrine_mongodb.odm.listeners.resolve_target_document');
         $this->assertDefinitionMethodCallOnce($definition, 'addResolveTargetDocument', [UserInterface::class, 'MyUserClass', []]);
-        $this->assertEquals(['doctrine_mongodb.odm.event_listener' => [['event' => 'loadClassMetadata']]], $definition->getTags());
+
+        if (in_array('Doctrine\Common\EventSubscriber',
+            class_implements($container->getParameter(trim($definition->getClass(), '%'))))) {
```

Comment:
Why would the test check that ? Assertions should not be conditional.

Reply:
Hi @stof actually when executing tests, the branch 1.1 of `doctrine/mongodb-odm` is used.
This branch do not actually have my modifications merged yet.
But I think you are right, I revert the changes and will wait until my 2 PRs are merged.
After the merge, the tests should be green

Reply:
I re-added the check. Since we are testing against lowest dependencies, we need to account for two separate versions where either one or the other tag will be added.